.gitignore

```diff
@@ -10,6 +10,11 @@ coverage/
 #
```

Comment:
I think the `.vscode` is also specific to the environment you are using as you might be using code, atom or any other editor.

Reply:
I agree. We already have a `.gitignore` on `master` but we should remove `.vscode` from there. Each developer should probably add `.vscode` to their global `.gitignore` or apply another strategy to ignore local files without adding them to `.gitignore`. See: https://help.github.com/en/github/using-git/ignoring-files

Reply:
@AfonsoVReis, please rebase with master and remove `.vscode` from `.gitignore` 🙏

Reply:
✅

Reply:
@AfonsoVReis `.DStore` is specific to macosx so that should be removed as well.

Reply:
✅thank you